house/tests/house_tests.py

```diff
@@ -36,6 +41,12 @@ def generate_house(generate_country, generate_city):
     return house
 
 
+@pytest.fixture
+def generate_get_request(request_factory):
```

Comment:
This fixture is unused. You can look at my PR https://github.com/redhat-beyond/Xpense/pull/71 to see the relevant test (test_board_function)

Reply:
thanks!
fixed